src/ResourceManager/Sql/Commands.Sql/ManagedInstance/Cmdlet/UpdateAzureSqlManagedInstance.cs

```diff
@@ -0,0 +1,212 @@
+// ----------------------------------------------------------------------------------
```

Comment:
Are there any parameters different between update and set? If not, you can remove "Set"

Reply:
Hm, not sure about this. As I understood Set will call PUT and any not specified parameter would be set to default. Update will however call PATCH and update only values that are specified through parameters

Reply:
Since New has put semantics, there's no need for the set if you also have a PATCH api call.

Reply:
Well New has put semantics, but New also checks if the managed instance exists, and fails if does. In discussion on previous closed PR, Mark Cowlishaw was for having both Set and Update, so that is why I added Set. Previously I had only Update